### PR TITLE
provide a couple of tips

### PR DESCRIPTION
**Pull request by kenburcham**
I gave just a couple of tips that would have been helpful to me!

### CODE REVIEW

File: Reference/Security/authenticate-with-AD.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ The `ActiveDirectoryBackOfficeUserPasswordChecker` will look in appSettings for
     <add key="ActiveDirectoryDomain" value="mydomain.local" />
 </appSettings>
 ```
+Note: one way to find your Active Directory Domain if you are logged into your domain is to open a command prompt and run "set logon" and use the value returned as the LOGONSERVER (not including any slashes).
```
Comment (sofietoft) [inline Reference/Security/authenticate-with-AD.md:67]:
```suggestion
:::tip
One way to find your Active Directory Domain if you are logged into your domain is to open a command prompt and run `set logon` and use the value returned as the `LOGONSERVER` (not including any slashes).
:::
```

**Review by sofietoft — COMMENTED** [review]
Thanks for adding the tips @kenburcham !
Makes a lot of sense!

I've suggested converting the note to a tip instead.
That way, it'll be rendered as a blue box with the tip inside it - let me know what you think!

**Review by sofietoft — APPROVED** [review]
Thanks for taking the suggestion! A box here, will make the tip stand out and almost impossible to miss 😄 

Thanks again for the PR! 
I can see that this is the first time you're contributing to the Umbraco Docs! That's awesome #H5YR 😎 
Do you have an account on [Our](https://our.umbraco.com/)? If you do, I can grant you a **C-Trib** badge for contributing. Let me know!

**Comment by kenburcham** [comment]
Yes, I’m on our: https://our.umbraco.com/members/kenburcham/

Thanks, I’d love a badge! ☺

**Comment by sofietoft** [comment]
Woohoo! You now have a new, shiny **C-Trib** badge on your Our profile 🎉 

Thanks again for helping out.
Looking forward to see more from you in the future!

**Pull request merged**